tests/node/rpc/messages/GetVehicleDataResponseTests.js

```diff
@@ -60,20 +79,21 @@ describe('GetVehicleDataResponseTests', function () {
 
     BaseRpcTests.tests();
 
-
     it ('testRpcValues', function (done) {
         let rpcMessage = this.msg;
         // Test Values
         const testStabilityControlsStatus = rpcMessage.getStabilityControlsStatus();
         const testHandsOffSteering = rpcMessage.getHandsOffSteering();
         const testWindowStatus = rpcMessage.getWindowStatus();
         const testGearStatus = rpcMessage.getGearStatus();
+        const testClimateData = rpcMessage.getClimateData();
 
         // Valid Tests
         Validator.assertEquals(this.stabilityControlsStatus, testStabilityControlsStatus);
         Validator.assertEquals(Test.GENERAL_BOOLEAN, testHandsOffSteering);
         Validator.assertEquals([Test.GENERAL_WINDOW_STATUS], testWindowStatus);
         Validator.assertEquals(this.gearStatus, testGearStatus);
+        Validator.assertEquals(this.climateData, testClimateData);
```

Comment:
Assuming you have setup `Test.js` with the `CLIMATE_DATA`, you can change line 96 to `Validator.assertEquals([TEST.GENERAL_CLIMATE_DATA], testClimateData);`.

Reply:
Fixed in 3881d93